app/assets/stylesheets/08-playlist-show.scss

```diff
@@ -75,7 +75,7 @@
     gap: 25px;
 }
 
-#playlist-play-button {
+#playlist-play-button, #artist-play-button {
```

Comment:
Do you need a separate class? Could you pull this into it's own component?

Reply:
I definitely could make a reusable component that would take different props depending on playlist VS artist... This is definitely not DRY, I just thought it would be faster for now. Ty.